src/operator/quantization/quantize_graph_pass.cc

```diff
@@ -156,6 +157,10 @@ Graph QuantizeGraph(Graph &&src) {
         if (avoid_quantize_input_map.count(node->op()) &&
             avoid_quantize_input_map[node->op()](node->attrs, i)) {
           new_node->inputs.emplace_back(mirror_entry);
+        // If network will directly accept quantized data, simply add data as input.
+        } else if (use_quantized_data_layer && e.node->is_variable() &&
+                   e.node->attrs.name == "data") {
```

Comment:
This is a very strong assumption on the name of input variable node. One solution is allowing users to specify the names of inputs and pass them as a graph attribute. By default, it can be called `data` if not provided. Need to document the usage well.

Reply:
Agree, this can be removed if  quantize_v2 can directly accept int8.